results.go

```diff
@@ -1281,6 +1286,72 @@ func (rh *ResultsHandler) uploadHandler(w http.ResponseWriter, r *http.Request)
 	http.Redirect(w, r, r.Referer(), http.StatusFound)
 }
 
+type combineResultsTemplateVars struct {
+	BaseTemplateVars
+
+	Results []SessionResults
+}
+
+const combinedSuffix string = "-combined"
```

Comment:
the type can be omitted here, since 'string' is inferred by the variable initialisation anyway